prefq/examples/imitation_preference_comparisons.py

```diff
@@ -145,11 +145,12 @@ def _send_videos_to_server(self, query_id):
 
         response = requests.post(self.server_url + "videos", files=payload, timeout=10)
 
-        if response.status_code >= 200 % response.status_code < 400:
+        try:
+            response.raise_for_status()
             print("PrefqGatherer: Payload transferred")
-        else:
-            print("PrefqGatherer: Error while sending videos")
-            print(response.status_code)
+        except requests.exceptions.HTTPError as err:
+            print(f"PrefqGatherer: Error while sending videos: {err}")
+            raise requests.exceptions.HTTPError
```

Comment:
You can just `raise err` to avoid losing information by re-creating the exception.